services/auth.js

```diff
@@ -138,6 +138,17 @@ const authService = {
     await User.updateOne({ _id: userId }, { $set: { password: hashedPassword } }).exec()
 
     await tokenService.removeResetToken(userId)
+  },
+
+  unsetFirstLogin: async (accessToken) => {
+    const tokenData = tokenService.validateAccessToken(accessToken)
```

Comment:
just `const { id: userId }...`

Reply:
Destructuring is done after the check (please see line 150)